### PR TITLE
Task5 testing

### PR DESCRIPTION
**Pull request by trumko**

### CODE REVIEW

File: src/components/ErrorBoundary/ErrorBoundary.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import ErrorBoundary from './ErrorBoundary'
+
+describe('ErrorBoundary', () => {
+  it('should render the warning if state.hasError is true', () => {
```
Comment (blyzniuk) [inline src/components/ErrorBoundary/ErrorBoundary.test.js]: You shouldn't depend on the realization in your test suite. The description should be clear even for not developer

`should render the warning when an error occurs`
`should render the regular content when there is no error`

File: src/utils/helpers.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { getMovieYear, shuffle } from './helpers';
+
+describe('getMovieYear', () => {
+  it('should properly extract the year from 2018-03-30', () => {
+    expect(getMovieYear('2018-03-30')).toMatchSnapshot();
```
Comment (blyzniuk) [inline src/utils/helpers.test.js]: As for me, snapshot here is redundant. Test is not only for testing. It is documentation also. Much easier to read test without snapshot. So prefer to use it only when you need to test equality with a big object/array or for rendered components
`expect(getMovieYear('2018-03-30')).toBe('2018');`

File: src/components/ErrorBoundary/ErrorBoundary.test.js (hunk 1 above)
Comment (trumko) [reply src/components/ErrorBoundary/ErrorBoundary.test.js]: fixed

File: src/utils/helpers.test.js (hunk 2 above)
Comment (trumko) [reply src/utils/helpers.test.js]: fixed, thank you

**Pull request merged**